dotnet/src/InternalUtilities/src/Data/VectorStoreRecordMapping.cs

```diff
@@ -5,93 +5,16 @@
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 using System.Linq;
-using System.Reflection;
 
 namespace Microsoft.Extensions.VectorData;
 
 /// <summary>
 /// Contains helper methods to map between storage and data models.
 /// </summary>
+// TODO: Kill this?
```

Comment:
I am not sure if "kill" is not too strong word to be included in a MS repo.

```suggestion
// TODO: Remove this?
```